src/client/particles.h

```diff
@@ -113,9 +113,9 @@ class Particle : public scene::ISceneNode
 
 	ClientEnvironment *m_env;
 	IGameDef *m_gamedef;
-	aabb3f m_box;
+	aabb3f m_box{v3f()};
```

Comment:
what's wrong with the default?

Reply:
The default does:
```cpp
aabbox3d(): MinEdge(-1,-1,-1), MaxEdge(1,1,1) {}
```
Whereas this here inits min and max with 0-vectors.

Well, it doesn't matter as `updateVertices()` is called anyway in the constructor.
Reverting the change.